lib/dialogs/firstUseReminder.dart

```diff
@@ -0,0 +1,258 @@
+                    onPressed: () => _set(context),
+                  ),
+                ),
+                ListTile(
+                  title: ButtonWidget(
+                    text: localizationUtil.translate("dont_set"),
+                    buttonType: ButtonType.Outline,
+                    color: Theme.of(context).primaryColor,
+                    onPressed: () => _no(context),
+                  ),
+                ),
+              ],
+            )
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future setNotification() async {
+    if (timeOfDay != null) {
+      FlutterLocalNotificationsPlugin flutterLocalNotificationsPlugin =
+      new FlutterLocalNotificationsPlugin();
+
+      // initialise the plugin. app_icon needs to be a added as a drawable resource to the Android head project
+      var initializationSettingsAndroid =
+      new AndroidInitializationSettings('ic_launcher');
+      var initializationSettingsIOS = new IOSInitializationSettings(
+          onDidReceiveLocalNotification: onDidReceiveLocalNotification);
+      var initializationSettings = new InitializationSettings(
+          initializationSettingsAndroid, initializationSettingsIOS);
+      flutterLocalNotificationsPlugin.initialize(initializationSettings,
+          onSelectNotification: onSelectNotification);
+
+      var time = Time(timeOfDay.hour, timeOfDay.minute, 0);
+      var androidPlatformChannelSpecifics = AndroidNotificationDetails(
+          'repeatDailyAtTime channel id',
+          'repeatDailyAtTime channel name',
+          'repeatDailyAtTime description');
+      var iOSPlatformChannelSpecifics = IOSNotificationDetails();
+      var platformChannelSpecifics = NotificationDetails(
+          androidPlatformChannelSpecifics, iOSPlatformChannelSpecifics);
+
+      await flutterLocalNotificationsPlugin.showDailyAtTime(
+          0,
+          localizationUtil.translate('did_you_daf'),
+          'd',
+          time,
+          platformChannelSpecifics);
+    } else {
+      toastUtil.showInformation(localizationUtil.translate("select_time"));
+    }
+  }
+
+  Future onSelectNotification(String payload) async {
+    if (payload != null) {
+      debugPrint('notification payload: ' + payload);
+    }
+//    await Navigator.push(
+//      context,
+//      new MaterialPageRoute(builder: (context) => new SecondScreen(payload)),
+//    );
+  }
+
+  // TODO: this function is from dialogs/firstUseDialogTwo. should have it in only one place.
+  DafLocationModel _getTodaysDaf() {
+    String today = DateFormat("MMMM d, y").format(DateTime.now());
+    return DafLocationModel.fromMap(datesBox.getDafForDate(today));
+  }
+
+  String _getTodaysDafString() {
+    DafLocationModel dafLocationModel = _getTodaysDaf();
+    String m = localizationUtil.translate(MasechetsData
+        .THE_MASECHETS[dafLocationModel.masechetId].translatedName);
+    String d = gematriaConverter.toGematria(dafLocationModel.dafIndex + 1);
+    return m + " " + d;
+  }
```

Comment:
see the change i did to `firstUseDialogOne.dart`.

```
  String getYesterdaysDaf() {
    DateTime yesterday =
        dateConverterUtil.getToday().subtract(Duration(days: 1));
    DafLocationModel yesterdaysDaf = dafsDatesStore.getDafByDate(yesterday);
    String masechet = localizationUtil.translate(yesterdaysDaf.masechetId);
    String daf = _getDafNumber(yesterdaysDaf.dafIndex);
    return masechet + " " + daf;
  }
```

```suggestion
  String _getTodaysDaf() {
    DateTime today =
        dateConverterUtil.getToday();
    DafLocationModel todaysDaf = dafsDatesStore.getDafByDate(today);
    String masechet = localizationUtil.translate(todaysDaf.masechetId);
    String daf = _getDafNumber(todaysDaf.dafIndex);
    return masechet + " " + daf;
  }
```